virtcontainers/qemu_arm64.go

```diff
@@ -189,3 +193,8 @@ func (q *qemuArm64) appendImage(devices []govmmQemu.Device, path string) ([]govm
 
 	return devices, nil
 }
+
+func (q *qemuArm64) setBypassSharedMemoryMigrationCaps(_ context.Context, _ *govmmQemu.QMP) error {
+	// bypass-shared-memory not support in arm64 for now
```

Comment:
cc @Pennyzct

Reply:
thanks for the heads-up, ;), yes, sadly, that's the current status.😥